package.json

```diff
@@ -14,21 +14,27 @@
   ],
   "license": "MIT",
   "main": "index.js",
+  "exports": {
```

Comment:
This addition makes this a breaking change no?  It cut's off access to `package.json`, right?

Reply:
Uh … technically, yes. I can add the “catch-all” forward, if this is important?

Reply:
I would prefer it, then we can remove it once we go to 1.x.

Reply:
ok, done! :)